static/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -57,7 +58,9 @@ function SentryAppExternalIssueForm({
             const queryParams = {referrer: appName};
             const url = addQueryParamsToExistingUrl(group.permalink, queryParams);
             const shortId = group.shortId;
-            return t('Sentry Issue: [%s](%s)%s', shortId, url, stackTrace);
+            return isFeedback
+              ? t('Sentry Feedback: [%s](%s)\n\n%s', shortId, url, group.metadata.message)
```

Comment:
Should this use more of the same format we see with github tickets? We could add the name + email address of who submitted it, and also info like whether a replay or attachment exists (all these should be loaded into the browser when the button is clicked)

Reply:
yeah, i can add email + name. right now GH doesn't have replay or attachment links in the ticket -- these descriptions are defined in the backend.

should we still add the replay link to this one?

Reply:
I'd put them all in here because it makes this template better, and that's where we should be headed anyway with all the templates.

With GH templates all on the backend we can followup with those changes as part of a separate ticket/pr (i think separate tickets already exist for that)

Reply:
don't know if i can get the replay link from this file, i can only access the replay ID from the event data

Reply:
<img width="526" alt="SCR-20240318-kouw" src="https://github.com/getsentry/sentry/assets/56095982/a9cd441f-c569-4096-b8e4-3e456d249cad">
<img width="807" alt="SCR-20240318-koxr" src="https://github.com/getsentry/sentry/assets/56095982/77f85f28-990c-404a-925b-dd2433642316">

this is what i have right now -- matches GH/jira

Reply:
check that the table syntax is the same, i think it might not be: https://linear.app/docs/markdown-editor

Reply:
i think the table markdown should be the same - it doesn't work without an empty header

Reply:
oh, it is the same! i notice now the screenshot with the dark background is from linear's rendering it :D
sorry for the confusion!